docs/README.md

```diff
@@ -42,8 +42,9 @@
-| [SRT Handshake](features/handshake.md)                       |     [features](features/)     | [handshake.md](features/handshake.md)                        | Description of SRT handshake mechanism. This<br />document might be outdated, please consult<br />[Section 3.2.1 Handshake](https://datatracker.ietf.org/doc/html/draft-sharabayko-srt-00#section-3.2.1) and<br />[Section 4.3 Handshake Messages](https://datatracker.ietf.org/doc/html/draft-sharabayko-srt-00#section-4.3) of the<br />[SRT RFC](https://datatracker.ietf.org/doc/html/draft-sharabayko-srt-00) additionally. |
+| [SRT Bonding: Main Backup][bonding-main-backup]              |     [features](features/)     | [bonding-main-backup.md][bonding-main-backup]                | SRT Main Backup Connection Bonding. |
+| [SRT Encryption](features/encryption.md)                     |     [features](features/)     | [encryption.md](features/encryption.md)                      | Description of SRT encryption mechanism. This<br />document might be outdated, please consult<br />[Section 6. Encryption][srt-rfc-encryption] of the [SRT RFC][srt-rfc] additionally. |
+| [SRT Handshake](features/handshake.md)                       |     [features](features/)     | [handshake.md](features/handshake.md)                        | Description of SRT handshake mechanism. This<br />document might be outdated, please consult<br />[Section 3.2.1 Handshake][srt-rfc-handshake] and<br />[Section 4.3 Handshake Messages](https://datatracker.ietf.org/doc/html/draft-sharabayko-srt-00#section-4.3) of the<br />[SRT RFC](https://datatracker.ietf.org/doc/html/draft-sharabayko-srt-00) additionally. |
 | [Live Streaming](features/live-streaming.md)<br />[Guidelines](features/live-streaming.md) |     [features](features/)     | [live-streaming.md](features/live-streaming.md)              | Guidelines for live streaming with SRT. See also<br />best practices and configuration tips in<br />[Section 7.1 Live Streaming](https://datatracker.ietf.org/doc/html/draft-sharabayko-srt-00#section-7.1) of the [SRT RFC](https://datatracker.ietf.org/doc/html/draft-sharabayko-srt-00). |
 | [SRT Packet](features/packet-filtering-and-fec.md)<br />[Filtering & FEC](features/packet-filtering-and-fec.md) |     [features](features/)     | [packet-filtering-and-fec.md](features/packet-filtering-and-fec.md) | Description of SRT packet filtering mechanism,<br />including FEC. |
 | [SRT Socket Groups](features/socket-groups.md)               |     [features](features/)     | [socket-groups.md](features/socket-groups.md)                | Description of socket groups in SRT (Connection<br />Bonding). Here you will also find the information<br />regarding the `srt-test-live` application for testing<br />Connection Bonding. |
```

Comment:
Could you please move this line in between "SRT Connection Bonding" and "SRT Bonding: Main Backup" so that we have connection bonding related documentation one after another.

Please rename "SRT Bonding: Main Backup" to "SRT Connection Bonding: Main/Backup".